packages/vite/package.json

```diff
@@ -1,18 +1,34 @@
 {
   "name": "vite",
   "version": "3.0.0-alpha.1",
+  "type": "module",
   "license": "MIT",
   "author": "Evan You",
   "description": "Native-ESM powered web dev build tool",
   "bin": {
     "vite": "bin/vite.js"
   },
-  "main": "dist/node/index.js",
-  "types": "dist/node/index.d.ts",
+  "main": "./dist/node/index.js",
+  "module": "./dist/node/index.js",
+  "types": "./dist/node/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/node/index.d.ts",
+      "import": "./dist/node/index.js",
+      "require": "./index.cjs"
+    },
+    "./client": {
+      "types": "./client.d.ts"
```

Comment:
This `./client` export is needed when using the new `nodenext` module resolution in the upcoming TypeScript 4.7 release.

As it stands now, using Vite `2.9.9` with typescript `4.7.1-rc` + `module: "nodenext"` does not work because the compiler cannot access `client.d.ts` as it is not explicitly exported.

```
/// <reference types="vite/client" />
```

Looking forward to this getting into the `3.0.0` release.